database/migrations/2018_01_13_184726_create_catalog_entities_table.php

```diff
@@ -25,7 +25,7 @@ public function up()
             $table->char('country_code', 2);
             $table->string('postal_code');
             $table->string('phone');
```

Comment:
Curious, why are those fields non-nullable, while the user's fields are nullable?

Reply:
To answer the question is that there isn't always a `catalogable entity` attached to the `user account` which effectively makes it a `fan account` (an account for purchasing and reviewing products etc) .. A `fan account` *DOESN'T* require all that *private/personal* information (but there if they want to for example add the information to receive promotional products or rewards like a t-shirt for spending so much money in a month type of thing) ... on the flip side a `catalogable entity` *DOES* require those fields for legal reasons, ensuring those constraints and maintaining `data integrity` *(TO NOT have a required field nullable for simplicity)* thats why you see 2 different "profiles" because they both have different purposes and constraints.